test/query.test.js

```diff
@@ -337,7 +338,7 @@ describe('query', () => {
 
     return client.query(query.Query(lambda)).then(function(body) {
       return client.query(body).then(function(bodyEchoed) {
-        expect(body).toEqual(bodyEchoed)
+        expect(body.value).toEqual(bodyEchoed.value)
```

Comment:
I had to update this assertion since `body` and `bodyEchoed` now both have `api_version` fields, which can mismatch. For now, we only really care about the `value` being equal.